requirements.txt

```diff
@@ -1,11 +1 @@
-enum34 ; python_version<='3.4'
-netaddr
-netifaces
-scapy>=2.4.3
-requests
-PrettyTable
-urllib3>=1.24.2,<1.25
-ruamel.yaml
-requests_mock
-future
-packaging
\ No newline at end of file
+.
```

Comment:
Make it editable so the original clone directory will be used instead of being copied (`-e .`)

Reply:
I am not sure if I understood this. Can you please put a reference link regarding this. I have never actually replaced the requirements file like this. Good to learn something new. Thanks!